src/Faker/Provider/kk_KZ/Person.php

```diff
@@ -183,7 +183,7 @@ class Person extends \Faker\Provider\Person
      */
     private static function getCenturyByYear($year)
     {
-        if ($year >= 2000 && $year <= DateTime::year()) {
+        if ($year >= 2000 && $year <= date('Y')) {
```

Comment:
why dont delete `&& $year <= date('Y')` ?

Reply:
To not run into a future I believe